apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -481,6 +496,20 @@ def render(element_html: str, data: pl.QuestionData) -> str:
             "Drag answer tiles into the answer area to the " + dropzone_layout + ". "
         )
 
+        if block_orientation is BlockOrientationType.HORIZONTAL:
+            if dropzone_layout != "bottom":
+                raise Exception(
+                    'The block-orientation attribute may only be "horizontal" when solution-placement is "bottom".'
+                )
+            if check_indentation:
+                raise Exception(
+                    'The indentation attribute may not be used when block-orientation is "horizontal".'
+                )
+            if max_indent != MAX_INDENTION_DEFAULT:
+                raise Exception(
+                    'The max-indent attribute may not be used when block-orientation is "horizontal".'
+                )
```

Comment:
I think this check is unnecessary because it is redundant--we already disallow having max-indent unless the indentation is on, which we are already disallowing with the check right before this.

Reply:
Removed.